pyproject.toml

```diff
@@ -12,6 +12,8 @@ license = "LICENSE"
 python = "^3.7"
 typing-extensions = ">=3.10.0.0"
 pydantic = "^1.6.2"
+redis = "^4.3.4"
```

Comment:
Please delete this. 

We don't want any extra dependency in core repo.

Reply:
Deleted